cpp/src/Glacier2CryptPermissionsVerifier/CryptPermissionsVerifierI.cpp

```diff
@@ -107,9 +107,11 @@ namespace
     {
 #if defined(__APPLE__) || defined(_WIN32)
 
+        // NOLINTBEGIN:cert-err58-cpp
```

Comment:
Can we just get rid of the constants? They're only used in this function. and we make a copy all the time.

Reply:
Fixed. In simple cases like these, we should use `const char* const`